Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASEIMAGE=us.gcr.io/k8s-artifacts-prod/build-image/debian-base-amd64:v2.1.0
+ARG BASEIMAGE=us.gcr.io/k8s-artifacts-prod/build-image/debian-base-amd64:v2.1.3
```

Comment:
Do we also need to bump the image `us.gcr.io/k8s-artifacts-prod/build-image/debian-iptables-amd64:v12.1.0` for NMI build?

Reply:
good catch, updated.